assignments/array-methods.js

```diff
@@ -56,28 +56,64 @@ const runners = [{"id":1,"first_name":"Charmain","last_name":"Seiler","email":"c
 // ==== Challenge 1: Use .forEach() ====
 // The event director needs both the first and last names of each runner for their running bibs.  Combine both the first and last names into a new array called fullName. 
 let fullName = [];
+
+runners.forEach(function(array){
+ fName = array.first_name;
+ lName = array.last_name;
+ fullName.push(fName);
+ fullName.push(lName);
```

Comment:
You get the idea here mechanically so it's not a big deal, but I believe the objective was to get an array of strings, with each string being the first name and the last name together.